tonic/src/codec/encode.rs

```diff
@@ -261,18 +206,20 @@ fn finish_encoding(
     Ok(())
 }
 
-#[derive(Debug)]
-enum Role {
+/// Role for use in encoding a body.
+#[allow(missing_docs)]
+#[derive(Clone, Copy, Debug)]
+pub enum Role {
```

Comment:
As long as there is no other `Role` (and similar usages in the rest of the API could potentially reuse this simple enum), I don't see much point in making the name more verbose/complicated. In rustls and Quinn we have this same enum but call it `Side` -- do you find that to be usefully more specific?

Reply:
> do you find that to be usefully more specific?

Yes. It seems like it would be hard to understand at a glance what this type is in a place where it's used. It may seem redundant, but I think the benefit of having a clear name outweighs it. We can change the name when importing it, but I think that would be more redundant in this case.

Reply:
Renamed to `Side`.